src/foam/nanos/cron/Cron.js

```diff
@@ -86,15 +87,25 @@ foam.CLASS({
       javaReturns: 'void',
       javaCode:
 `DAO notification = (DAO) x.get("notificationDAO");
-    
+            
 Notification cronStartNotify = new Notification();
-cronStartNotify.setBody("Cron job: " + this.getId() + " started");
+StringBuilder startMsg = new StringBuilder();
+startMsg.append("Cron job: ").append(this.getId()).append(" started.");
+if ( ! SafetyUtil.isEmpty(this.getDescription()) ) {
```

Comment:
Ok, this is messy.
how about: 
Cron STARTED - id description 
Cron ENDED - id description

Reply:
Actually I vote for @jlhughes's suggestion above  than what I suggested below as well.

Above is even more simpler -- easier to do grepping on, etc.